roles/docker/templates/docker-svc.j2

```diff
@@ -12,7 +12,7 @@ ExecStart=/usr/bin/docker daemon --storage-opt dm.thinpooldev=/dev/mapper/contiv
 {% else %}
 ExecStart=/usr/bin/docker daemon -H fd:// --cluster-store=etcd://localhost:{{ etcd_client_port1 }}
 {% endif %}
-MountFlags=slave
+MountFlags=shared
```

Comment:
@mapuri shall we make this a ansible variable with defaults to `shared`? for cases where people are not using k8s or volplugin and where using `slave` should be ok. wdyt?